### PR TITLE
 	new file:   setup.py

### PR DESCRIPTION
**Pull request by seasidesparrow**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        return default
+
+setup(
+    name='ingest_data_model',
+    version=get_git_version(default="v0.0.1"),
+    url='http://github.com/adsabs/ingest_data_model/',
+    license='MIT',
+    author='NASA/SAO ADS',
+    description='JSON Schema for ADS Ingest Data',
+    long_description=__doc__,
```
Comment (kelockhart) [inline setup.py:30]: I don't actually know what this line would do (which docstring is it pointing to?) - I'd hardcode a string here, or, better, do something like they do here and point to the README: https://packaging.python.org/en/latest/tutorials/packaging-projects/#configuring-metadata (see the example code block in this section)

**Review by kelockhart — CHANGES_REQUESTED** [review]
Looks good! Just one minor change and then you can merge.

File: setup.py (hunk 1 above)
Comment (seasidesparrow) [reply setup.py]: Ok, this is fixed.

**Review by kelockhart — APPROVED** [review]
Looks good!

**Pull request merged**